File: FFRv2/app/src/main/java/com/devingotaswitch/rankings/domain/appsync/tags/PostHypeSleeperTag.java
package com.devingotaswitch.rankings.domain.appsync.tags;

import com.devingotaswitch.utils.Constants;

import java.util.Arrays;
import java.util.HashSet;

public class PostHypeSleeperTag extends Tag {
    public PostHypeSleeperTag(int count) {
        super(count, Constants.POST_HYPE_SLEEPER_TAG, new HashSet<>(Arrays.asList(Constants.QB, Constants.RB, Constants.WR, Constants.TE)));
    }
}

File: FFRv2/app/src/main/java/com/devingotaswitch/rankings/domain/appsync/tags/PostHypeSleeperTag.kt
package com.devingotaswitch.rankings.domain.appsync.tags

import com.devingotaswitch.utils.Constants
import java.util.*

class PostHypeSleeperTag(count: Int) : Tag(count, Constants.POST_HYPE_SLEEPER_TAG, Constants.POST_HYPE_SLEEPER_TAG_REMOTE, HashSet(listOf(Constants.QB, Constants.RB, Constants.WR, Constants.TE)))